src/ContainerRegistry/ContainerRegistry/Models/ContainerRegistryDataPlaneClient.cs

```diff
@@ -123,7 +123,7 @@ private string GetRefreshToken()
                     .RefreshTokenProperty;
         }
 
-        private string GetAccessToken(string scope)
+        public string GetAccessToken(string scope)
```

Comment:
Do you need to change the scope of this method?

Reply:
> Do you need to change the scope of this method?

Cli provided a way to expose this access token in cmdlet, though we do not have it right now. It's possible we need it in the future.

Reply:
Please public it when you use it. Don't add too much in one PR.